File: Algorithm/Backjun/bronze5/b10951/b10951.js
// A+B -4
// https://www.acmicpc.net/problem/10951

const readline = require('readline')

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
})

let input = []

rl.on('line', (line) => {
  input.push(line)
})

const getSum = (a, b) => {
  return a + b
}

rl.on('close', () => {
  let results = []
  for (let i in input) {
    const [a, b] = input[i].split(' ').map(Number)
    results.push(getSum(a, b))
  }

  results.forEach(results => console.log(results))
})

File: Algorithm/Backjun/bronze5/Star_b10951/b10951.c
// A+B -4
// https://www.acmicpc.net/problem/10951

#include <stdio.h>
#include <stdlib.h>

// 두 수를 더하는 함수
int getSum(int a, int b) {
    return a + b;
}

int main() {
    int a, b;
    int n = 0;
    int *results = (int *)malloc(n * sizeof(int));

    while (scanf("%d %d", &a, &b) != EOF) {
        results[n++] = getSum(a, b);
    }

    for (int i = 0; i < n; i++) {
        printf("%d\n", results[i]);
    }

    free(results);

    return 0;
}
